crates/holochain/src/conductor/manager/mod.rs

```diff
@@ -295,6 +356,13 @@ fn handle_completed_task(kind: &TaskKind, result: ManagedTaskResult, name: Strin
             Ok(_) => LogInfo(name),
             Err(err) => MinorError(Box::new(err), name),
         },
+        TaskKind::Persistent => match result {
+            Ok(_) => LogInfo(name),
+            Err(err) => {
+                tracing::warn!("Restarting persistent tasks is not yet implemented.");
+                MinorError(Box::new(err), name)
```

Comment:
Should this be fatal for now, if we're depending on the task being persistent but we don't have the restart mechanism yet?